From 2ee525f014bb42059a629cca6e2a3464f9c3861a Mon Sep 17 00:00:00 2001
From: Justin Carpentier <justin.carpentier@inria.fr>
Date: Wed, 9 Sep 2020 14:46:50 +0200
Subject: [PATCH 2/2] cmake: sync submodule

---
 cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule cmake be9051c..cc6f687:
diff --git a/cmake/python.cmake b/cmake/python.cmake
index db414f9..2a9fb82 100644
--- a/cmake/python.cmake
+++ b/cmake/python.cmake
@@ -175,7 +175,11 @@ MACRO(FINDPYTHON)
   MESSAGE(STATUS "PythonLibraryDirs: ${PYTHON_LIBRARY_DIRS}")
   MESSAGE(STATUS "PythonLibVersionString: ${PYTHONLIBS_VERSION_STRING}")
 
-  IF(NOT PYTHON_SITELIB)
+  IF(PYTHON_SITELIB)
+    IF(WIN32)
+      STRING(REGEX REPLACE "\\" "/" PYTHON_SITELIB "${PYTHON_SITELIB}")
+    ENDIF(WIN32)
+  ELSE(PYTHON_SITELIB)
     # Use either site-packages (default) or dist-packages (Debian packages) directory
     OPTION(PYTHON_DEB_LAYOUT "Enable Debian-style Python package layout" OFF)
     # ref. https://docs.python.org/3/library/site.html
@@ -203,7 +207,7 @@ MACRO(FINDPYTHON)
     IF(PYTHON_PACKAGES_DIR)
       STRING(REGEX REPLACE "(site-packages|dist-packages)" "${PYTHON_PACKAGES_DIR}" PYTHON_SITELIB "${PYTHON_SITELIB}")
     ENDIF(PYTHON_PACKAGES_DIR)
-  ENDIF(NOT PYTHON_SITELIB)
+  ENDIF(PYTHON_SITELIB)
 
   MESSAGE(STATUS "Python site lib: ${PYTHON_SITELIB}")
 
-- 
2.21.1 (Apple Git-122.3)

